.tool-versions

```diff
@@ -1 +1 @@
-nodejs 18.16.1
+nodejs 21.6.1
```

Comment:
Use an old version to check renovate works maybe?

Reply:
https://github.com/NotMyFault/incrementals-publisher/issues/6

Applied this PR as patch https://github.com/NotMyFault/incrementals-publisher/commit/f9a1e7bff75117ce74bd22d2155db141c3f9c5e5 and renovate detects it in its dashboard

Reply:
Actually, I removed the regex check after renovate picked it up as intended in the issue linked above.

Reply:
Is this possible to keep an LTS line to avoid bad surprises?